src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -64,6 +64,13 @@
         },
         "Bundle": {
             "EntryLimit": 500
+        },
+        "Throttling": {
+            "Enabled": true,
+            "ConcurrentRequestLimit": 100,
```

Comment:
This seems highly dependency on the environment, should it just be off be default?